src/LiveComponent/src/LiveComponentHydrator.php

```diff
@@ -553,6 +558,14 @@ private function hydrateObjectValue(mixed $value, string $className, bool $allow
             return new $className($value);
         }
 
+        if (is_a($className, AbstractUid::class, true)) {
+            if (!\is_string($value)) {
+                throw new BadRequestHttpException(\sprintf('The model path "%s" was sent an invalid data type "%s" for a uuid.', $propertyPathForError, get_debug_type($value)));
+            }
+
+            return $className::fromString($value);
+        }
```

Comment:
Since hydrate/de-hydrate a Symfony's AbstractUid means that you already it in your app (and so, have `symfony/uid` in your dependencies), we do not need to check `class_exists(AbstractUid::class)` right?

Reply:
I think so